### ISSUE TITLE
Gene counts on tab should be updated if facet filters are set

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/monarch-legacy#1270]
Otherwise we end up with confusing situations like this:

<img width="1412" alt="screen shot 2016-06-07 at 11 15 19 am" src="https://cloud.githubusercontent.com/assets/50745/15869454/4fa5dd9a-2ca1-11e6-86a8-7eea7440e287.png">

**Comment by jmcmurry** [comment]
I agree this is confusing. I wonder though whether updating the tab could make it appear artificially sparse. Might it be preferable to document the denominator better?
Eg. Perhaps we could instead display "showing 1-1 selected from 10,457"?

**Comment by jmcmurry** [comment]
Related to https://github.com/monarch-initiative/monarch-app/issues/1263
